driver-core/src/main/com/mongodb/binding/SingleConnectionReadBinding.java

```diff
@@ -64,6 +65,11 @@ public SessionContext getSessionContext() {
         return NoOpSessionContext.INSTANCE;
     }
 
+    @Override
+    public BsonTimestamp getClusterTime() {
+        return null;
```

Comment:
Better to throw UnsupportedOperationException here.  It's not like there isn't a cluster time available.  We just haven't done the work to plumb it through yet, and currently there is no need.

Reply:
Done.